lib/config/specification.yml

```diff
@@ -919,6 +938,7 @@ time_entries:
     - user_id # the internal ID of the user who created this time entry
     - approved # whether or not the time entry has been approved
     - role_id # The internal ID of the Role that is associated with this time entry
+    - integrated_resource # Object representing the status of integration synchronization
```

Comment:
When we use this gem, I think we are going to send in something like this
```
user_id: 123,
integrated_resource: {service_name: 'netsuite', service_model: 'time_bill'}
```

integrated_resource is a nested attribute